packages/fast-components-react-base/src/typography/typography.props.ts

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+import { IManagedClasses, ITypographyClassNameContract } from "@microsoft/fast-components-class-name-contracts";
+
+export enum TypographyTag {
+    caption = "caption",
+    h1 = "h1",
+    h2 = "h2",
+    h3 = "h3",
+    h4 = "h4",
+    h5 = "h5",
+    h6 = "h6",
+    p = "p",
+    span = "span"
+}
+
+export enum TypeLevel {
+    _1 = 1,
+    _2 = 2,
+    _3 = 3,
+    _4 = 4,
+    _5 = 5,
+    _6 = 6,
+    _7 = 7,
+    _8 = 8,
+    _9 = 9
+}
+
+export interface ITypographyHandledProps {
+    /**
+     * The component children
```

Comment:
We may want to re-word the comments for these. What we are really saying (correct me if I am wrong) is that this is the heading content, which is not limited to text-only.

Reply:
i can replace with content